### PR TITLE
Check if the session is interactive before prompting for user input

### PR DESCRIPTION
**Pull request by BSchilperoort**
See #151

### CODE REVIEW

File: era5cli/utils.py [hunk 1]
```diff
@@ -211,14 +212,17 @@ def strtobool(value: str) -> bool:
...
-            "\n  Do you want to overwrite them? (Y/N)"
-            "\n  Tip: to skip this flag, use `--overwrite`."
-            "\n"
-        )
+        answer = "no"  # default answer for non-interactive sessions
+        if sys.stdin.isatty():  # only ask for input if the user can reply.
+            answer = input(
+                "\n  Some file(s) that will be downloaded already exist in this folder."
+                "\n  Do you want to overwrite them? (Y/N)"
+                "\n  Tip: to skip this flag, use `--overwrite`."
```
Comment (sverhoeven) [inline era5cli/utils.py:220]: Can you move the tip above the question. Now the cursor to type the answer is not close to the question

File: era5cli/utils.py [hunk 2]
```diff
@@ -211,14 +212,17 @@ def strtobool(value: str) -> bool:
...
+                "\n"
+            )
         if answer.lower() in ["n", "no", "nope"]:
```
Comment (sverhoeven) [inline era5cli/utils.py:223]: If 'answer' is '' (just press enter) then file is overwritten. I was expecting no as the default.

**Review by sverhoeven — CHANGES_REQUESTED** [review]
Tested with `ssh localhost era5cli ...` and it does not prompt.

File: era5cli/utils.py (hunk 1 above)
Comment (sverhoeven) [reply era5cli/utils.py:220]: Also `skip this flag` is confusing to me
```suggestion
                "\n  Tip: to always overwrite use `--overwrite`."
```

**Comment by BSchilperoort** [comment]
> Tested with `ssh localhost era5cli ...` and it does not prompt.

Thanks for your review! I have taken into account your comments.

**Review by sverhoeven — APPROVED** [review]
Looks good to me.

**Pull request merged**